install-chromedriver.sh

```diff
@@ -1,38 +1,58 @@
 #!/usr/bin/env bash
 
 echo """
-Installer script to set up chromedriver for selenium
-on WSL. This allows for running selenium automated
-browser tests.
+Installer script to set up chromedriver for selenium.
+First installs some linux dependencies for this script.
+Then installs google-chrome-stable from google repository.
+Uses that version of chrome to install chromedriver into PATH.
+This should work for linux systems to run headless selenium
+tests against chrome.
 
-Needs approximately the following in the bash environment:
+On WSL, however, installing chrome this way does not work in
+headless mode. You can still run the selenium tests by pointing
+to the chrome executable path from Windows.
+
+Note:
+    WSL environment should look something like:
```

Comment:
Might be worth adding that the env is expected to be a `Debian` based distro.

Reply:
This was mainly my collected notes for when hopping between machines. Some of it is needed for CI and I suppose it's a useful developer tool for local selenium development.